diff -rupN RKTrackRep/RKTrackRep.cxx TMPRKTrackRep/RKTrackRep.cxx
--- RKTrackRep/RKTrackRep.cxx	2010-10-12 17:37:53.000000000 +0200
+++ TMPRKTrackRep/RKTrackRep.cxx	2010-10-12 18:34:11.000000000 +0200
@@ -33,8 +33,8 @@
 #include"TMath.h"
 #include"TGeoManager.h"
 #include"TDatabasePDG.h"
-#include"GFException.h"
-#include"GFFieldManager.h"
+#include"core/GFException.h"
+#include"core/GFFieldManager.h"
 
 #define MINSTEP 0.001   // minimum step [cm] for Runge Kutta and iteration to POCA
 
diff -rupN RKTrackRep/RKTrackRep.h TMPRKTrackRep/RKTrackRep.h
--- RKTrackRep/RKTrackRep.h	2010-10-12 17:38:06.000000000 +0200
+++ TMPRKTrackRep/RKTrackRep.h	2010-10-12 18:35:10.000000000 +0200
@@ -29,10 +29,10 @@
 #define RKTRACKREP_H
 
 
-#include "GFAbsTrackRep.h"
-#include "GFDetPlane.h"
+#include "core/GFAbsTrackRep.h"
+#include "core/GFDetPlane.h"
 
-#include "geometry/GFMaterialEffects.h"
+#include "core/geometry/GFMaterialEffects.h"
 
 /** @brief Track Representation module based on a Runge-Kutta algorithm including a full material model
  *
